### PR TITLE
Reddit com

### PR DESCRIPTION
**Pull request by VolhaHoman**
Steps
4.	Go to messages
5.	Delete “Welcome to reddit” message
can't be added due to inability to find a workaround for the captcha.

### CODE REVIEW

File: src/main/resources/story/reddit/CreateAccountReddit.story [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Scenario: Open reddit.com main page
+Meta:
+   @test
+Given I am on the main application page
+
+Scenario: Create user account
+Meta:
+   @test
+When I refresh the page
```
Comment (apaulenkova) [inline src/main/resources/story/reddit/CreateAccountReddit.story:11]: Is this step required?

File: src/main/resources/story/reddit/CreateAccountReddit.story [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+!-- When I decline an action in an alert with the message 'www.reddit.com wants to Show notifications'
+When I click on element located `By.xpath(//a[text()='sign up'])`
+When I switch to a frame by the xpath '//iframe[contains(@src,'actionSource')]'
+When I change context to an element by the xpath '//div[@class='Container']'
+When I enter '<email>' in a field by the xpath '//input[@id='regEmail']'
+When I click on a button with the name 'Next'
+When I select a random username
+When I change context to an element by the xpath '//div[contains(@class,'usernameStep')]'
+When I enter '<password>' in a field by the xpath '//input[@id='regPassword']'
```
Comment (apaulenkova) [inline src/main/resources/story/reddit/CreateAccountReddit.story:19]: All these input fields have more friendly values under attributes 'name' or 'placeholder', so you can use them

File: src/main/resources/story/reddit/CreateAccountReddit.story [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+When I switch to a frame by the xpath '//iframe[contains(@name,'a-')]'
+When I check a checkbox by the xpath '//div[@class='recaptcha-checkbox-border']'
+!-- re-CAPTCHA for humans
+When I click on a button with the name 'Sign Up'
+!-- When I click on an element by the xpath '//button[contains(@class,'SignupButton')]'
+Then the text 'Find communities by topics you're interested in.' exists
+When I click on a button with the name 'Finish'
+!-- When I click on an element by the xpath '//button[contains(@class,'SubscribeButton')]'
+When I refresh the page
+Then an image with the src 'https://www.redditstatic.com/avatars/avatar_default_20_FF4500.png' exists
```
Comment (apaulenkova) [inline src/main/resources/story/reddit/CreateAccountReddit.story:29]: I think that the absolute path to this img can change after a time.
I think that relative would be more reliable: Then an image with the src containing '$srcpart' exists

File: src/main/resources/story/reddit/CreateAccountReddit.story (hunk 1 above)
Comment (VolhaHoman) [reply src/main/resources/story/reddit/CreateAccountReddit.story:11]: @apaulenkova when the main page opens, appears a request to send notifications.  The best workaround, I thought that moment, was to refresh the page. I couldn't even inspect this element.

**Pull request merged**